### DISCUSSION TITLE
Request for the visually impaired organist

### DISCUSSION THREAD

**Discussion by newVideophile** [GrandOrgue/grandorgue#1473] [Developers]
May I ask the developers for a change in the code that should be minor, but still very helpful for my ageing eyes:
![ageing organist](https://user-images.githubusercontent.com/95248680/231133789-a0daf552-bb52-450f-9180-ed1789920740.jpg)

**Comment by larspalo** [comment]
Well, we have had globally adjustable font sizes earlier that didn't work out so well. A possible way would be to allow a font scaling for individual elements instead, but for now, until such a feature is available, what you want can easily be achieved with a custom panel in the odf. I attach an example of just that. You'll possibly need to adjust the [Panel001] value (also for each element in the panel) to whatever number is available next in the odf (and remember to increase the NumberOfPanels by one) before you paste it into any .organ file.

[LargerSequencer.txt](https://github.com/GrandOrgue/grandorgue/files/11200882/LargerSequencer.txt)

Also, don't forget that you can easily scale up the whole panel if you want to by changing the window size.

**Comment by newVideophile** [comment]
Tank you very much for your answer! I'm not familiar with odf file manipulation, but it I will try what you suggest.

The window scaling method is what I use currently, but it is neither elegent nor convenient as the Combination setter window position is not remembered, so I have to reposition it each time I start GrandOrgue. I have mentioned this issue in another topic already (which I am going to update now).

**Reply by larspalo** [reply]
If you need help with creating your custom panel in any odf, just ask and I'll be happy to provide more details.

**Comment by oleg68** [comment]
I made some enlargement of the font in #1475

See here: https://github.com/GrandOrgue/grandorgue/pull/1475/files#diff-fe41cb503b0cb0d8b080e8073f04b8ffe27d5fbc6ba060bbbfdf6379ac7fd067
